modules/xact/connection/ssh/mdmimpl/SSHNETCONFConnectionImpl/build.xml

```diff
@@ -36,6 +36,9 @@
         <ant antfile="build.xml" target="build" inheritAll="false" dir="${basedir}/../SSHConnectionImpl">
             <property name="target.dir" value="${basedir}/lib/xyna" />
         </ant>
+        <ant antfile="build.xml" target="build" inheritAll="false" dir="${basedir}/../../sharedlib/SSHJSchUtils">
```

Comment:
the directory name "SSHJschUtils" should be renamed, because it doesn't contain anything regarding Jsch any more?

Reply:
Rename SSHJSchUtils to SSHJUtils